src/fsharp/FSharp.Core/prim-types.fsi

```diff
@@ -715,7 +715,7 @@ namespace Microsoft.FSharp.Core
         new : unit -> NoDynamicInvocationAttribute
 
         [<Experimental("Experimental library feature, requires '--langversion:preview'")>]
```

Comment:
Because this is internal, we don't need the `Experimental` attribute.

Reply:
done thanks